cmd/reissuer/main.go

```diff
@@ -404,6 +415,15 @@ func reissueCerts(c *cli.Context) (err error) {
 		certreq.DnsNames = append(certreq.DnsNames, dnsNames...)
 	}
 
+	// Override the certificate delivery webhook if specified
+	var webhook string
+	if webhook = c.String("webhook"); webhook != "" {
+		certreq.Webhook = webhook
+	}
```

Comment:
Creating a variable in a single lexical block gives the compiler more opportunities for optimization. 

```suggestion
	if webhook := c.String("webhook"); webhook != "" {
		certreq.Webhook = webhook
	}
```